src/nupic/engine/Network.hpp

```diff
@@ -148,22 +189,17 @@ class Link;
    * @returns A pointer to the newly created Region
    */
   std::shared_ptr<Region> addRegion(const std::string &name,
-  					const std::string &nodeType,
-                    const std::string &nodeParams);
+  					                        const std::string &nodeType,
+                                    const std::string &nodeParams);
 
     /**
-     * Create a new region in a network from serialized region
+     * Add a region in a network from deserialized region
```

Comment:
nit "from a serialized region"

Reply:
> nit "from a serialized region"

Hmmm, not very clear is it.   This is a region that has been deserialized by itself and we want to add it to the network.